composer.json

```diff
@@ -11,22 +11,25 @@
             "email": "dcramer@gmail.com"
         }
     ],
-    "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2.1",
-        "phpunit/phpunit": "^4.8 || ^5.0",
-        "ext-openssl": "*",
-        "nokitakaze/testhttpserver": "^0.1.0",
-        "monolog/monolog": "~1.0"
-    },
     "require": {
-        "php": ">=5.5",
-        "ext-hash": "*",
+        "php": ">=5.6",
```

Comment:
>For HHVM, I don't know if I would still be testing on that, since a lot of other big players are leaving it

I was following a thread some weeks ago that was talking about this, and it seems that it's true that many of the big libraries/frameworks out of here are dropping HHVM due to the lack of big performance differences since PHP 7 was released and since there are some different behaviours between the two platforms that makes maintaining the code more difficult, so I'm 👍 on dropping support. I would simply not test it anymore, as it's useless to have it on `allow_failure` if we don't support it